schemas/v1.0/azure.yaml.json

```diff
@@ -437,6 +437,15 @@
                     }
                 }
             ]
+        },
+        "cloud": {
+            "type": "object",
+            "title": "The cloud configuration used for the project.",
+            "description": "Optional. Provides additional configuration for deploying to sovereign clouds such as Azure Government.",
+            "additionalProperties": false,
+            "properties": {
+                "name": { "enum": [ "AzureCloud", "AzureChinaCloud", "AzureUSGovernment" ] }
```

Comment:
Feels like if the user is going to specify the cloud section then the `name` should be required.